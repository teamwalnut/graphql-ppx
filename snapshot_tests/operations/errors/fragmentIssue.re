[%graphql
  {|
  fragment Screen_Model on Screen {
    id
  }

  fragment DemoVersion_Model on DemoVersion {
    screens {
      ...Screen_Model
    }
  }
|}
];
