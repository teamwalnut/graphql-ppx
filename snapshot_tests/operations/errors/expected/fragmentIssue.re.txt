
  [1;31mWe've found a bug for you![0m
  [36msnapshot_tests/operations/errors/fragmentIssue.re[0m:[2m8:5-10:5[0m

   6 [2mâ”‚[0m 
   7 [2mâ”‚[0m   fragment DemoVersion_Model on DemoVersion {
   [1;31m8[0m [2mâ”‚[0m     [1;31mscreens {[0m
   [1;31m9[0m [2mâ”‚[0m [1;31m      ...Screen_Model[0m
  [1;31m10[0m [2mâ”‚[0m [1;31m    }[0m
  11 [2mâ”‚[0m   }
  12 [2mâ”‚[0m |}

  This type constructor, `Screen_Model.Raw.t_DemoScreen`, can't be found.
  

